### PR TITLE
Add contributing guidelines

### PR DESCRIPTION
**Pull request by nglgzz**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+First off, thank you for considering contributing to Learn Anything. It's people
+like you that make Learn Anything such a great platform (and community).
+
+The following guidelines help communication between maintainers of the project,
+avoid wasted effort, and should make it as easy as possible for you to make the
+changes that you want to see.
+
+There are many ways you can contribute to this project: improving documentation
+(including these guidelines), submitting bug reports, or writing code that will
+be incorporated in the site.
```
Comment (nikitavoloboev) [inline CONTRIBUTING.md]: I like integrated more than incorporated here.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+2. What was the expected outcome?
+3. What happened instead?
+4. Add any additional details that could be useful.
+
+
+### 3. Assign Issue
+You decided to work on an issue:
+- If there is already someone assigned, please don't work on it. Rather comment
+and ask how you can help.
+- If no one is assigned to it, assign yourself.
```
Comment (nikitavoloboev) [inline CONTRIBUTING.md]: I believe only maintainers can assign people to issues. Maybe should be mentioned?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+If the changes you make are scoped to only one part of the stack, include that
+in the commit message. A couple of examples:
+
+```
+GraphQL: add studyPlans query
+FE: refactor search page
+```
+
+### 5. Pull Request
+Pull requests (PR) should be targeting the `latest` branch.
```
Comment (nikitavoloboev) [inline CONTRIBUTING.md]: Can comment on what branches will new LA have? Will current 'master' branch be renamed to 'latest' and 'dev' branch is going away?

**Comment by nikitavoloboev** [comment]
I think for issues we should use [GitHub templates](https://github.com/yoshuawuyts/github-templates). 

I linked to a small tool that helps you make them. What do you think?

File: CONTRIBUTING.md (hunk 3 above)
Comment (nglgzz) [reply CONTRIBUTING.md]: `latest` is temporary until we have something to deploy. Then we'll get rid of other branches and only keep master. And maybe an extra branch for github pages if we'll still need that.

File: CONTRIBUTING.md (hunk 2 above)
Comment (nglgzz) [reply CONTRIBUTING.md]: Okay didn't know, I'll change this part then. What do you think would be a good way to do this? People just comment saying that they're going to work on issues and we assign them?

Comment (nikitavoloboev) [reply CONTRIBUTING.md]: Yes. I think that would be best.

**Comment by nglgzz** [comment]
Yeah, I like that, I will add GitHub templates as well.

**Comment by nikitavoloboev** [comment]
Looks awesome. ✨

Let's merge this?

**Pull request merged**